index.html

```diff
@@ -39,11 +43,135 @@ <h1>Nsong Fitz Gerald</h1>
                 <div class="button">
                     <button id="introbutton">Start collaboration</button>
                 </div>
+            </div> 
+        </section>
+        <section>
+            <div id="titleimg"></div>
+            <div id="projecttitle">
+                <h1 class="title">Projects</h1>
             </div>
-
         </section>
-       
-
+    <section id="projects">
```

Comment:
- [ ] The first reviewer highlighted that you should align elements in the parent container of the project section (`<section id="projects">`) section using grid. 
https://github.com/fitz95/Portfolio-Microverse-/pull/3#discussion_r1038763491

It is a requirement for this milestone. Kindly implement this change. 👍